docker-bake.hcl

```diff
@@ -0,0 +1,108 @@
+variable "DOCKER_REGISTRY" {default = "ghcr.io/openclarity"}
+variable "DOCKER_TAG" {default = "latest"}
+variable "SUFFIX" {default = ""}
+
+variable "VERSION" {default = null}
+variable "BUILD_TIMESTAMP" {default = null}
+variable "COMMIT_HASH" {default = null}
+variable "VMCLARITY_TOOLS_BASE" {default = null}
+
+function "get_tag" {
+  params = [name]
+  result = ["${DOCKER_REGISTRY}/${name}${SUFFIX}:${DOCKER_TAG}"]
+}
+
+group "default" {
+	targets = ["vmclarity-apiserver", "vmclarity-cli", "vmclarity-cr-discovery-server", "vmclarity-orchestrator", "vmclarity-ui", "vmclarity-ui-backend"]
+}
+
+group "e2e" {
+	targets = ["vmclarity-apiserver", "vmclarity-cli", "vmclarity-orchestrator", "vmclarity-ui", "vmclarity-ui-backend"]
+}
+
+target "_common" {
+	labels = {
+		"org.opencontainers.image.url" = "https://github.com/openclarity/vmclarity"
+		"org.opencontainers.image.licenses" = "Apache-2.0"
+	}
+	cache-to = ["type=local,dest=/tmp/.buildx-cache"]
+	cache-from = ["type=local,src=/tmp/.buildx-cache"]
+	output = ["type=image"]
+}
+
+target "_common_args_for_go" {
+  args = {
+		VERSION = "${VERSION}"
+		BUILD_TIMESTAMP = "${BUILD_TIMESTAMP}"
+		COMMIT_HASH = "${COMMIT_HASH}"
+  }
+}
+
+target "vmclarity-apiserver" {
+	context = "."
+	dockerfile = "Dockerfile.apiserver"
+	tags = get_tag("${target.vmclarity-apiserver.name}")
+	inherits = ["_common", "_common_args_for_go"]
+	labels = {
+		"org.opencontainers.image.title" = "VMClarity API Server"
+		"org.opencontainers.image.description" = "The VMClarity API for managing all objects in the VMClarity system."
+	}
+}
+
+target "vmclarity-cli" {
+	context = "."
+	dockerfile = "Dockerfile.cli"
+	tags = get_tag("${target.vmclarity-cli.name}")
+	inherits = ["_common", "_common_args_for_go"]
+	args = {
+		VMCLARITY_TOOLS_BASE = "${VMCLARITY_TOOLS_BASE}"
```

Comment:
Default `make docker-cli` complains that `ERROR: failed to solve: base name (${VMCLARITY_TOOLS_BASE}) should not be blank` which overwrites the default value specified as ARG in CLI docker image.
Commenting these args resolves the error.